docs/guide/data.md

```diff
@@ -43,25 +43,25 @@ Example:
 ]
 ```
 
-This information can be helpful when interpreting the data that is returned or for narrowing down the data requested to only the columns necessary or selected.
+This information can be helpful when interpreting the returned data or for narrowing down the requested data to only the necessary/selected columns.
 
 ## Requesting Data
 
-The [data](https://api-user.arable.cloud/api/v2/doc#operation/get_data) endpoint provides access to time series data from a specified table. It takes the name of the table to query as a path parameter, and returns data in JSON or CSV format depending on the value of an optional `Accept` header (the default is JSON).
+The [data](https://api-user.arable.cloud/api/v2/doc#operation/get_data) endpoint provides access to time series data from a specified table. It takes the name of the table to query as a path parameter, and returns data in JSON or CSV format depending on the value of an optional `accept` header (the default is JSON).
```

Comment:
This should be capital Accept

Reply:
OK. Do I change the edit, or do you reject it?

Reply:
you can change the edit.